### PR TITLE
MOCKS: Payouts index and empty state

### PR DESCRIPTION
**Pull request by philschanely**
## Description

In support of Commerce team work on Kajabi Payments Onboarding and Payouts, this PR completes mocks for the Payouts index page and empty states in Rails.

## Screenshots

Screen Shot 2022-05-05 at 12 31 43 PM

Screen Shot 2022-05-05 at 2 28 52 PM

## Testing in `sage-lib`

See http://localhost:4000/pages/mock/kp_onboarding_payouts

## Testing in `kajabi-products`

1. N/A - Mocks for Payouts Index and empty state

## Related

https://kajabi.atlassian.net/browse/SAGE-515

### CODE REVIEW

File: docs/app/views/mocks/kp_onboarding_payouts/payouts/_empty_state.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <% end %>
+
+  <% content_for :sage_empty_state_actions do %>
+    <%= sage_component SageButton, {
+      attributes: {
+        # TODO: Dev needs to wire in the bank setup process here before going to this confirm modal.
+        "data-js-modaltrigger": "modal-payouts-bank-config-confirm"
+      },
+      style: "primary",
+      value: "Link bank account",
```
[the remark this answers is not in the document]
Comment (philschanely) [reply docs/app/views/mocks/kp_onboarding_payouts/payouts/_empty_state.html.erb]: @monicawheeler fixed by adding a Button Group to set the desired gap between them.

**Review by monicawheeler — APPROVED** [review]
💯

File: docs/app/views/mocks/kp_onboarding_payouts/payouts/_modal_payout_schedule.html.erb [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<%= sage_component SageModal, { id: "modal-payout-schedule" } do %>
```
[the remark this answers is not in the document]
Comment (pixelflips) [reply docs/app/views/mocks/kp_onboarding_payouts/payouts/_modal_payout_schedule.html.erb:1]: @philschanely just saw the comments below. Just resolve and ignore this one if you're aware.

**Review by pixelflips — APPROVED** [review]
LGTM! Really nice work here!! 👍🏼

**Pull request merged**